.github/CODEOWNERS

```diff
@@ -630,6 +630,7 @@
 
 
 # Frontend unit tests
+/core/templates/components/entity-creation-services/story-creation.service.spec.ts @nithusha21 @U8NWXD
```

Comment:
I don't think we need this change as `/core/templates/**/*.spec.ts @nithusha21` on line 634 should cover this*

Is the pre-push-checks are failing without this line?

Reply:
Yes! according to @taiwo2